.ci/compile.sh

```diff
@@ -145,12 +134,10 @@ if [[ $(uname) == "Darwin" ]]; then
   flags+=("-DCMAKE_PREFIX_PATH=${qt_location[0]}")
 fi
 
-if [[ $USE_CCACHE ]]; then
-  flags+=("-DUSE_CCACHE=1")
-  if [[ $CCACHE_SIZE ]]; then
-    # note, this setting persists after running the script
-    ccache --max-size "$CCACHE_SIZE"
-  fi
+# Add cmake --build flags
+buildflags=(--config "$BUILDTYPE")
+if [[ $PARALLEL_COUNT ]]; then
+  buildflags+=(--parallel "$PARALLEL_COUNT")
```

Comment:
--parallel was added in CMake 3.12, but our minimum is 3.10. This causes Bionic to fail

Reply:
right, I knew this and forgot

Reply:
I've added a workaround